services/common/src/components/permits/PermitConditionForm.tsx

```diff
@@ -307,6 +316,18 @@ const PermitConditionForm: FC<PermitConditionFormProps> = ({
                                                 : "Add Report Requirement"}
                                         </Button>
                                     </Col>
+                                    {enablePermitConditionTags && conditionTags &&
+                                        <Col className="condition-tag-select">
+                                            <Field
+                                                name="condition_tags"
```

Comment:
I'm wondering, since this is another item added to a row of a bunch of buttons, how wide does your screen have to be to keep it all in one line? Does it deserve its own line?

Reply:
I think it looks decent on smaller screens as is.  It wraps when things get smaller so it never seems to look worse than anything else considering we're not generally optimized for mobile.

<img width="804" height="807" alt="image" src="https://github.com/user-attachments/assets/6b8a8798-e353-44c0-a666-24377c20bdb3" />